### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by Ivordir**
Fixes a typo in the README and makes other spelling corrections in the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This version of Palette has been automatically tested with Rust version `1.60.0`
 
 ## Contributing
 
-All sorts of contributions are welcome (<strike>including</strike> especially speling corrections), so take a look at [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines, if you are interested.
```
Comment (Ogeon) [inline README.md:22]: This one is intentional 😁

**Review by Ogeon — COMMENTED** [review]
Good catches! Thank you! I would like to keep "speling" around for the joke, tough 😅

File: README.md (hunk 1 above)
Comment (Ivordir) [reply README.md]: So it was intentional! :satisfied:

**Comment by Ivordir** [comment]
Btw, I used https://github.com/crate-ci/typos to find these spelling mistakes.

**Comment by Ogeon** [comment]
But, thank you! With the intentionally misspelled word back where it belongs, I think this is good to go.

bors r+

**Pull request merged**